cde-pentaho/resource/js/cdf-dd-base.js

```diff
@@ -269,7 +269,7 @@ var SynchronizeRequests = {
 
     $.post(wd.cde.endpoints.getPluginUrl() + "Syncronize", saveSettingsParams, function(result) {
       try {
-        var json = eval("(" + result + ")");
+        var json = JSON.parse(result);

```

Comment:
that code is dead. It's irrelevant whether there are changes there or not.
In fact, we should remove the whole cde-pentaho folder.